### PR TITLE
chore(repo): update stalebot ignored labels

### LINKED ISSUE #3356: chore(repo): update stalebot ignored labels
Add `enhancement` and `Feature` labels to the list of labels the stalebot tool will ignore, so that these issues never get marked stale & automatically closed out.

### PR DESCRIPTION
**Pull request by evwilkin**
closes #3356 

Add enhancement & Feature labels to stalebot ignore

### CODE REVIEW

**Comment by mcoker** [comment]
Users often add these labels to issues they create since they're generic - do we want to allow for that? What if we had something like "design approved" that could be applied to an enhancement or feature issue (and maybe other things).

Also, scope creep, but can we also make these changes?
* remove "pinned" and "security" since core doesn't have those
* update the breaking changes label to "breaking changes" - we don't have the 💥

**Comment by evwilkin** [comment]
@mcarrano follow-up from our earlier discussion - I'd like to get your feedback on @mcoker's points above on using a `design approved` label in place of adding `enhancement` and `Feature` labels to the ignore list for stalebot.

**Comment by mcarrano** [comment]
@evwilkin @mcoker I'm OK with whatever works for you guys.

**Comment by evwilkin** [comment]
@mcoker updated per your feedback above

File: .github/stale.yml [hunk 1]
```diff
@@ -8,10 +8,9 @@ daysUntilClose: 30
 
 # Issues with these labels will never be considered stale
 exemptLabels:
-  - pinned
-  - security
   - accessibility
-  - "breaking change :boom:"
+  - "breaking changes"
```
Comment (mcoker) [inline .github/stale.yml:12]: double check me, but pretty sure this is the label....

```suggestion
  - "breaking change"
```

**Comment by evwilkin** [comment]
@mcoker you're right, updated from `breaking changes` to `breaking change`

File: .github/stale.yml [hunk 2]
```diff
@@ -8,10 +8,9 @@ daysUntilClose: 30
 
 # Issues with these labels will never be considered stale
 exemptLabels:
-  - pinned
-  - security
   - accessibility
-  - "breaking change :boom:"
+  - "breaking change"
+  - design approved
```
Comment (mcoker) [inline .github/stale.yml:13]: do we need quotes? `"breaking change"` has quotes but `design approved` doesn't.

Comment (evwilkin) [reply .github/stale.yml:13]: There should not be quotes - removed these

**Pull request merged**